zappa/zappa.py

```diff
@@ -435,14 +438,13 @@ def splitpath(path):
     ##
 
     def upload_to_s3(self, source_path, bucket_name):
-        """
+        r"""
```

Comment:
For consistency, always use """triple double quotes""" around docstrings. Use r"""raw triple double quotes""" if you use any backslashes in your docstrings. For Unicode docstrings, use u"""Unicode triple-quoted strings""" . 

https://www.python.org/dev/peps/pep-0257/